We do not package onxruntime which is a dependency of piper
For more details see https://bugs.gentoo.org/961974#c1

diff --git a/setup/extensions.json b/setup/extensions.json
index b39ce6d..c105031 100644
--- a/setup/extensions.json
+++ b/setup/extensions.json
@@ -134,14 +134,6 @@
         "error": "!podofo_error",
 		"needs_c++": "17"
     },
-    {
-        "name": "piper",
-        "sources": "calibre/utils/tts/piper.cpp",
-		"needs_c++": "17",
-        "libraries": "!piper_libs",
-        "lib_dirs": "!piper_lib_dirs",
-        "inc_dirs": "!piper_inc_dirs"
-    },
     {
         "name": "html_as_json",
         "sources": "calibre/srv/html_as_json.cpp",
